docs/java/how-to-develop-a-workflow-definition-in-java.md

```diff
@@ -50,19 +47,93 @@ public interface FileProcessingWorkflow {
 }
 ```
 
+### `@WorkflowMethod`
+
+**@WorkflowMethod** denotes the starting point of [Workflow Execution](/docs/concepts/what-is-a-workflow-execution). Workflow Execution completes when this method completes.
+A Workflow interface can define only a single method annotated with `@WorkflowMethod`.
 The `@WorkflowMethod` annotation has a `name` parameter, such as `@WorkflowMethod(name = "MyWorkflowType")`.
-It can be used to denote the [Workflow Type](/docs/concepts/what-is-a-workflow-type). If not set, the Workflow Type defaults to the short name of the Workflow interface. In the example above, the Workflow Type defaults to `FileProcessingWorkflow`.
+It can be used to denote the [Workflow Type](/docs/concepts/what-is-a-workflow-type). If not set, the Workflow Type defaults to the short name of the Workflow interface. In the previous example, the Workflow Type defaults to `FileProcessingWorkflow`.
 A method annotated with `@WorkflowMethod` can have any number of parameters. We recommend passing a single parameter that contains all the input fields. This allows adding fields in a backward compatible manner.
```

Comment:
This should move up to be after line 53 - before the Workflow Type info - and allow the Workflow Type info + example to flow.

Reply:
done